xgi/drawing/draw.py

```diff
@@ -887,6 +885,8 @@ def draw_node_labels(
     clip_on_nodes : bool, optional
         Turn on clipping of node labels at axis boundaries.
         By default, True.
+    zorder : int, optional
+        The layer on which to draw the nodes.
```

Comment:
The vertical order on which to draw the labels. 
If None (default) the labels are drawn on the top level.